### ISSUE TITLE
GUI : FED : LDAP Accessibility Violations

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#2755]
## Overview
The LDAP page has many accessibility violations that need to be addressed. There is another story for addressing the table check boxes.

### Tasks
- [ ] Run the IBM Accessibility Checker on the LDAP page and resolve the fieldset and double label issues
![image](https://user-images.githubusercontent.com/1183515/98825337-f9d13b80-23f9-11eb-9a75-8e64b5df1a23.png)

## UX Flow / Interaction Requirements
N/A

## Final Invision Mockup
N/A

## cURL Commands
N/A

### Browser Tests (Chrome, Firefox, Safari (Mac), Edge (Windows))
- [ ] Tested responsive layout on supported viewport sizes
- [ ] Verified functionality works as expected, e.g. success and failure use cases
- [ ] Resolved any JavaScript errors thrown to the console

### Accessibility Tests
- [ ] Verified markup is semantic
- [ ] No reported issues in IBM Accessibility Checker (Free open source tool) https://www.ibm.com/able/toolkit/tools/

**Comment by yoshiemuranaka** [comment]
Resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/38337
